.github/workflows/test.yml

```diff
@@ -65,4 +65,50 @@ jobs:
         python -m pytest -v --durations=0 -x test_trainable.py
         declare -a arr=("AsyncHyperBandScheduler" "HyperBandScheduler" "MedianStoppingRule" "ASHAScheduler"); for s in "${arr[@]}"; do python schedulers.py --scheduler "$s"; done
         cd ../examples
-        for f in *.py; do echo "running $f" && python "$f" || exit 1 ; done
\ No newline at end of file
+        for f in *.py; do echo "running $f" && python "$f" || exit 1 ; done
+
+  build_docs:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        git clone https://github.com/ray-project/ray
+        python -m pip install -U -r ray/doc/requirements-rtd.txt
+        python -m pip install -r ray/doc/requirements-doc.txt
+    - name: Install package
+      run: |
+        python -m pip install -e .
+    - name: Build the documentation
+      run: |
+        export DOC_BUILD=1 PYTHONWARNINGS=ignore
+        cd ray/doc/ && sphinx-build -q -E -W -T -b html source _build/html
+
+  test_lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Do we need to fetch the entire repo for this? The latest commit should be good. I'd just remove it and have it use all default arguments, line in `build_docs`.

Reply:
`format.sh` uses the merge base; so checking out 1 commit wasn't sufficient. The repo is small though.

Reply:
Gotcha, right.